CMakeLists.txt

```diff
@@ -517,7 +517,7 @@ set ( ALSA_MINIMUM_VERSION 0.9.1 )
 set ( DBUS_MINIMUM_VERSION 1.11.12 )
 set ( GLIB2_MINUMUM_VERSION 2.6.5 )
 set ( LIBINSTPATCH_MINIMUM_VERSION 1.1.0 )
-set ( LIBSNDFILE_MINIMUM_VERSION 1.0.0 )
+set ( LIBSNDFILE_MINIMUM_VERSION 1.2.1 )
```

Comment:
Why did you bump the minimum version here?

Reply:
In [libsndfile release 1.2.1](https://github.com/libsndfile/libsndfile/releases/tag/1.2.1) the CMake fixes contributed by @FtZPetruska were published, so I think it makes sense to use it here as a minimum.

I've also tried to find some fix for `sf_open()` on the source tree, but without luck. 😞

Reply:
Ah, ok, that's fine then.